src/goto-instrument/full_slicer.cpp

```diff
@@ -322,18 +329,38 @@ Function: implicit
 
 \*******************************************************************/
 
-static bool implicit(goto_programt::const_targett target)
+static bool implicit(const namespacet &ns, goto_programt::const_targett target)
 {
   // some variables are used during symbolic execution only
```

Comment:
This function now seems to be doing a lot more than what it originally was intended for!